### DISCUSSION TITLE
The Otel Collector release process, how does it work?

### DISCUSSION THREAD

**Discussion by stephenhong** [open-telemetry/opentelemetry-collector#6891] [Q&A]
Hello team, I'm interested in learning more about the release process of the otel collector. I see in [this](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/release.md) documentation that "build and testing is currently fully automated." May I ask how the testing process is structured? If possible, I would like to set up this automated build and testing process locally and see how it runs

**Comment by jpkrohling** [comment]
The contributing guide shows how to run the tests locally. We use GitHub Actions to run the tests automatically.

https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md
